src/legacy/core_plugins/console/np_ready/public/lib/ace_token_provider/token_provider.ts

```diff
@@ -0,0 +1,91 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { IEditSession, TokenInfo } from 'brace';
+import { TokensProvider, Token, Position } from '../../interfaces';
+
+// Brace's token information types are not accurate.
+interface ActualTokenInfo extends TokenInfo {
```

Comment:
Why not give it the name `TokenInfo` and rename the brace once `TokenInfo as BraceTokenInfo` when importing?